### ISSUE TITLE
HTML view: add option to show data on per virtual host basis

### ISSUE THREAD

**Issue by bonanza123** [allinurl/goaccess#605]
Please correct me if I'm wrong, but currently the HTML view shows e.g. the browser or OS statistics for all (virtual) hosts. It would be nice if e.g. in the "panel options" there would be the opportunity to show the stats on demand only for user-chosen virtual hosts.

**Comment by allinurl** [comment]
You are correct. However, issue #117 will address this and give the ability to filter by virtual host or any other field.
